src/System.IO.FileSystem.AccessControl/src/Resources/Strings.resx

```diff
@@ -188,4 +188,16 @@
   <data name="Arg_PathEmpty" xml:space="preserve">
     <value>The path is empty.</value>
   </data>
+  <data name="ArgumentOutOfRange_NeedPosNum" xml:space="preserve">
+    <value>Positive number required.</value>
+  </data>
+  <data name="Argument_InvalidAppendMode" xml:space="preserve">
+    <value>Append access can be requested only in write-only mode.</value>
+  </data>
+  <data name="Argument_InvalidFileModeAndAccessCombo" xml:space="preserve">
+    <value>Combining FileMode: {0} with FileAccess: {1} is invalid.</value>
+  </data>
+  <data name="NotSupported_FileStreamOnNonFiles" xml:space="preserve">
+    <value>FileStream was asked to open a device that was not a file. For support for devices like 'com1:' or 'lpt1:', call CreateFile, then use the FileStream constructors that take an OS handle as an IntPtr.</value>
+  </data>
```

Comment:
What's actually referring to this resource?

Reply:
Same case, I ended up removing it.
It was originally being consumed by the code I copied from FileStream in coreclr, which is why I added them in the first place. But as I cleaned the code, these resources were not called anymore, and I forgot to remove them from the resx.
Thanks for catching them.

Reply:
Turns out I needed this one but not the one for FileAccess. Instead I added a new one for FileSystemRights.